### PR TITLE
Update ICSI -> CS&S

### PR DESCRIPTION
**Pull request by choldgraf**
Updates our public documentation to list CS&S as our fiscal sponsor.

### CODE REVIEW

File: reference/terminology.md [hunk 1]
```diff
@@ -3,12 +3,25 @@
 Here are some helpful terms that we use at 2i2c.
 
 ```{glossary}
-Source of Truth
-Single Source of Truth
-  Distributed teams and open communities need to balance information across team members, and ensure that everyone is on the same page. For this reason, it is recommended to adopt a "single source of truth" for anything important. This is an authoritative source that everyone can look to in order to know the current status and plan for anything we do at 2i2c.
+CS&S
+Code for Science and Society
```
Comment (damianavila) [inline reference/terminology.md:7]: It is nice to see you can assign two terms to the same description!

**Review by damianavila — APPROVED** [review]
LGTM

File: code-of-conduct/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Our goal is to create one of the best communities in the world for learning, using, and creating open infrastructure for interactive computing. Our mission is to build this community in a way that promotes fairness and justice for all - especially those from traditionally marginalized groups. This allows every member of our community to thrive and to make the biggest positive impact in their work and on others. Our Code of Conduct defines expected behavior and guidelines that help create such a community.
 
-Accordingly, anyone who participates in a 2i2c space is expected to show respect and courtesy to others in all interactions, whether in GitHub repositories, our Slack channel, in in-person events, when representing 2i2c in public, or in events and spaces associated with [ICSI](structure:icsi).
+Accordingly, anyone who participates in a 2i2c space is expected to show respect and courtesy to others in all interactions, whether in GitHub repositories, our Slack channel, in in-person events, when representing 2i2c in public, or in events and spaces associated with {term}`CS&S`.
```
Comment (sgibson91) [inline code-of-conduct/index.md:6]:
```suggestion
Accordingly, anyone who participates in a 2i2c space is expected to show respect and courtesy to others in all interactions, whether in GitHub repositories, our Slack channel, during in-person events, when representing 2i2c in public, or in events and spaces associated with {term}`CS&S`.
```

Just found the repeated "in" a little jarring

**Review by sgibson91 — APPROVED** [review]
🎉

**Comment by choldgraf** [comment]
thanks all for the review!

**Pull request merged**